expression/builtin_math_vec_test.go

```diff
@@ -23,7 +23,9 @@ import (
 )
 
 var vecBuiltinMathCases = map[string][]vecExprBenchCase{
-	ast.Sign: {},
+	ast.Sign: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt}},
```

Comment:
The type of its child should be `ETReal`.

Reply:
done